hw/ip/prim/doc/prim_lfsr.md

```diff
@@ -70,18 +69,15 @@ LFSR with the `DefaultSeed` in the next cycle.
 
 The LFSR coefficients are taken from an internal set of lookup tables with
 precomputed coefficients. Alternatively, a custom polynomial can be provided
-using the `Custom` parameter. The lookup tables contain polynomials for both
-LFSR forms and range from 4bit to 64bit for the Galois form and 3bit to 168bit
-for the Fibonacci form. The polynomial coefficients have been obtained from
-[this page](https://users.ece.cmu.edu/~koopman/lfsr/) and
+using the `Custom` parameter. The lookup table contains polynomials for both
+LFSR forms and range from 3bit to 168bit.
+The polynomial coefficients have been obtained from
 [Xilinx application note 52](https://www.xilinx.com/support/documentation/application_notes/xapp052.pdf).
 The script `./script/get-lfsr-coeffs.py` can be used to download, parse and dump
 these coefficients in SV format as follows:
 ```
 $ script/get-lfsr-coeffs.py -o <output_file>
 ```
-The default is to get the Galois coefficients. If the Fibonacci coefficients
-are needed, add the `--fib` switch to the above command.
```

Comment:
Can you adapt the script accordingly as well? Or if we choose to keep the other coefficient source in there, can we make `fib` the default option?

Reply:
I think there's value in keeping both coefficient sources - e.g. if a user wants to provide custom coefficients, download from [Koopman](https://users.ece.cmu.edu/~koopman/lfsr/) will provide multiple polynomials to choose from.
For `prim_lfsr.sv` I've selected the Xilinx source as a default because it provides a wider range for LFSR width ( 3 to 168, rather than 4 to 64)

I have now added a commit that modifies `get-lfsr-coeffs.py` to remove references to lfsr_type and to rename `--fib` switch with `--pdf`. I'm not sure how to make the Xilinx source into default because it requires a pdf file at the input.
The way I have it now, if the pdf file path is provided the script will use it, otherwise it will default to downloading from  [Koopman](https://users.ece.cmu.edu/~koopman/lfsr/)